consensus/core/src/mass/mod.rs

```diff
@@ -55,3 +55,35 @@ pub fn transaction_output_estimated_serialized_size(output: &TransactionOutput)
     size += output.script_public_key.script().len() as u64;
     size
 }
+
+pub struct MassCalculator {
```

Comment:
the duplication of this struct within consensus/src should be removed

Reply:
Oh, this is a mishap on my end.  This was done so long time ago so I have since assumed that this is a part of the original codebase. 

However, `wallet/core/src/tx/mass.rs` has a new implementation that needs to be migrated to consensus and perhaps merged with `consensus/src/processes/mass.rs`.  

Removed this instance of `MassCalculator`, aiming to address the merge in a separate PR.